pkg/render/manager.go

```diff
@@ -674,6 +685,33 @@ func managerClusterRole(managementCluster, managedCluster, usePSP bool) *rbacv1.
 				},
 				Verbs: []string{"get", "create"},
 			},
+			{
+				// Add access to Linseed APIs.
+				APIGroups: []string{"linseed.tigera.io"},
+				Resources: []string{
+					"flows",
+					"flowlogs",
+					"bgplogs",
+					"auditlogs",
+					"kube_auditlogs",
+					"ee_auditlogs",
```

Comment:
Do we also need any kube logs ?

Reply:
"auditlogs" is any audit logs - it's two lines up.